.github/workflows/puppeteer.yml

```diff
@@ -18,7 +18,7 @@ env:
   IMGUR_CLIENT_ID: ${{ secrets.IMGUR_CLIENT_ID }}
   TERM: xterm-256color
   JDK_CURRENT: 11.0.12
-  SCENARIO_REGEX: ".*" # Use this to limit which tests run
+  SCENARIO_REGEX: "db.*" # Use this to limit which tests run
```

Comment:
Something to note/remove before the final merge. I assume this is temporary for now to speed things up :)